packages/website-eslint/src/index.js

```diff
@@ -24,8 +25,27 @@ exports.esquery = esquery;
 
 exports.createLinter = function () {
   const linter = new Linter();
-  for (const name in rules) {
-    linter.defineRule(`@typescript-eslint/${name}`, rules[name]);
+  for (const name in plugin.rules) {
+    linter.defineRule(`@typescript-eslint/${name}`, plugin.rules[name]);
   }
   return linter;
 };
+
+/** @type {Record<string, unknown>} */
+const configs = {};
+
+for (const name in eslintJs.configs) {
+  configs[`eslint:${name}`] = eslintJs.configs[name];
+}
+
+for (const name in plugin.configs) {
+  const value = plugin.configs[name];
```

Comment:
[Nitpick]

```suggestion
for (const [name, value] of Object.entries(plugin.configs)) {
```

I wonder if we should enable this: https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/prefer-object-from-entries.md

Reply:
from computation point of view my code is faster 🕙 as we don't have too loop trough this 2 times :)